screens/questions/Questions.css

```diff
@@ -0,0 +1,122 @@
+/* main container */
+.question-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+/* section box */
+.question-box {
+  width: 32.5rem;
+  border-radius: 1.2rem;
+  box-shadow: 0 5px 20px -5px rgb(106, 109, 109);
```

Comment:
You can use CSS variable over here. Just a suggestion. 
Everything else looks good to me. Nice UI! Good to merge👍

Reply:
Thanks, I will do that!